src/gpuarray_blas_cuda_cublas.c

```diff
@@ -520,6 +520,25 @@ static int hgemmBatch(cb_order order, cb_transpose transA, cb_transpose transB,
                       gpudata **B, size_t *offB, size_t ldb,
                       float beta, gpudata **C, size_t *offC, size_t ldc,
                       size_t batchCount) {
+  (void)order;
+  (void)transA;
+  (void)transB;
+  (void)M;
+  (void)N;
+  (void)K;
+  (void)alpha;
+  (void)A;
+  (void)offA;
+  (void)lda;
+  (void)B;
+  (void)offB;
+  (void)ldb;
+  (void)beta;
+  (void)C;
+  (void)offC;
+  (void)ldc;
+  (void)batchCount;
+
```

Comment:
@abergeron It's hideously ugly but AFAIK portable to MSVC too. Your alternatives are to
1) Create a compiler-specific `#define UNUSED __attribute__((unused))` and stick that on every parameter and hope MSVC has something like it that goes in the same position (But I think MSVC's solution is to simply not write the name of the argument, which GCC rejects as "parameter name omitted"),
2) Use a flag like  `-Wnounused-parameter` after `-Wall -Wextra` to disable the warning
3) Use compiler-specific `pragma`'s equivalent to `-Wnounused-parameter`.